powerbi-docs/enterprise/service-premium-scale-out.md

```diff
@@ -68,6 +68,7 @@ By default, scale-out is enabled for your tenant, but it's not enabled for seman
     | App, library or service  | Version |
     |--------------------------|---------|
     | Microsoft Analysis Services OLE DB Provider for Microsoft SQL Server (MSOLAP) | 16.0.20.201 (March 2022) |
+    | Analyze in Excel for Power BI Semantic models (MSOLAP) | 16.0.122.27 |
```

Comment:
nit: Models for case-consistency?

Reply:
Power BI docs use the term, "Power BI semantic models" (small s, small m)